app/views/thredded/private_topics/_breadcrumbs.html.erb

```diff
@@ -2,3 +2,8 @@
   <li><%= link_to t('thredded.nav.all_messageboards'), messageboards_path -%></li>
   <li><%= link_to t('thredded.nav.private_topics'), private_topics_path -%></li>
 </ul>
+
+<ul class="thredded--navigation-breadcrumbs">
+  <li><%= link_to t('thredded.nav.mark_all_read'), mark_all_read_private_topics_path, { method: :put } -%></li>
+</ul>
+
```

Comment:
As far as where to put this - what about somewhere that's directly related to, and within, the private topics section where it's not too too far up in peoples' faces. Maybe at the bottom of the page 1 listing?

![](https://www.evernote.com/l/AUp_kioVt6ZECrUmYZzCEh68UiyK5zbhXcUB/image.png)